prefect_docker/credentials.py

```diff
@@ -9,6 +9,24 @@
 class DockerRegistryCredentials(Block):
     """
     Block used to manage credentials for interacting with a Docker Registry.
+
+    Examples:
+        Log into Docker Registry.
+        ```python
+        from prefect import flow
+        from prefect_docker import DockerHost, DockerRegistryCredentials
+
+        @flow
+        def docker_registry_login_flow():
+            docker_host = DockerHost()
+            docker_registry_credentials = DockerRegistryCredentials(
+                username="my_username",
+                password="my_password",
+                registry_url="registry.hub.docker.com",
+            )
+            with docker_host.get_client() as client:
+                docker_registry_credentials.login(client)
```

Comment:
This doesn't need to be in a flow, so the minimal example might be more clear if we don't run it in a flow.
```suggestion
        from prefect_docker import DockerHost, DockerRegistryCredentials

        docker_host = DockerHost()
        docker_registry_credentials = DockerRegistryCredentials(
            username="my_username",
            password="my_password",
            registry_url="registry.hub.docker.com",
        )
        with docker_host.get_client() as client:
            docker_registry_credentials.login(client)
```